src/utils/lang-utils.ts

```diff
@@ -34,3 +34,22 @@ export const loadIncontextTranslation = (
     document.head.appendChild(jipt);
   }
 };
+
+/**
+ * Filter out unsupported languages and return an Object containing language code and language name
+ * @param excludedLanguages
+ * @returns Object containing language code and language name
+ */
+export const getAllowedLanguages = (
+  excludedLanguages: Omit<LanguageCode, "ACH">[] = []
```

Comment:
where does this hardcoded ACH come from?

Reply:
Its a key present in [src/utils/constants.ts](https://github.com/deriv-com/translations/pull/41/files#diff-e8d0a358ae2fb090c56f00a058050983cc9691b7d3622ddd443170017123e7d9)